src/js/themes/base.js

```diff
@@ -384,6 +384,13 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         width: `${borderWidth}px`,
         radius: `${baseSpacing * 0.75}px`,
       },
+      disabled: {
+        border: {
+          // color: { dark: undefined, light: undefined }
+        },
```

Comment:
I think you're right. Let me do that.

Reply:
Actually, I believe border needs to remain because we check for border.color in disabledStyle. If it's commented it out it fails:
<img width="378" alt="Screen Shot 2020-04-10 at 9 34 22 AM" src="https://user-images.githubusercontent.com/12522275/79006731-a2d75100-7b0e-11ea-9191-9e9fc8d6dc04.png">

Reply:
Was it already in the existing behavior? If its new behavior, we should check for `border` before we check for `border.color`

Reply:
Gotcha, it's new. I will update to check for `border` first. Then I will comment the whole object out in the theme.

Reply:
Done. : )